<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-JA0T08A" failures="0" tests="1" name="Default test" time="0.08" errors="0" timestamp="2022-11-11T11:12:46 MEZ">
  <testcase classname="testcases.BaseClass" name="@BeforeTest DataSetUp" time="0.008">
    <failure type="java.io.FileNotFoundException" message="exceldata.xlsx (Das System kann die angegebene Datei nicht finden)">
      <![CDATA[java.io.FileNotFoundException: exceldata.xlsx (Das System kann die angegebene Datei nicht finden)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at testcases.BaseClass.DataSetUp(BaseClass.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:645)
at org.testng.TestRunner.beforeRun(TestRunner.java:634)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest DataSetUp -->
  <testcase classname="testcases.BaseClass" name="@BeforeMethod SetUp" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod SetUp -->
  <testcase classname="testcases.BaseClass" name="@BeforeMethod SetUp" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod SetUp -->
  <testcase classname="testcases.BaseClass" name="@AfterMethod TearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase classname="testcases.BaseClass" name="@AfterTest DataClean" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest DataClean -->
  <testcase classname="testcases.SimplilearnLoginTest" name="Test1" time="0.0">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase classname="testcases.SimplilearnLoginTest" name="Test3" time="0.0">
    <ignored/>
  </testcase> <!-- Test3 -->
  <testcase classname="testcases.SimplilearnLoginTest" name="Test2" time="0.0">
    <ignored/>
  </testcase> <!-- Test2 -->
</testsuite> <!-- Default test -->
